scripts/zones/Metalworks/npcs/Cid.lua

```diff
@@ -53,7 +52,7 @@ entity.onTrigger = function(player, npc)
 
     if wsQuestEvent ~= nil then
         player:startEvent(wsQuestEvent)
-    elseif (currentCOPMission == tpz.mission.id.cop.DAWN and player:getCharVar("PromathiaStatus")==3 and player:getCharVar("Promathia_kill_day")~=currentday and player:getCharVar("COP_tenzen_story")== 0 ) then
+    elseif (currentCOPMission == tpz.mission.id.cop.DAWN and player:getCharVar("PromathiaStatus")==3 and player:getCharVar("Promathia_kill_day")<os.time() and player:getCharVar("COP_tenzen_story")== 0 ) then
```

Comment:
can you reformat this line please? its wide and the operators have no spacing around them.